.github/workflows/swan-ci-ca.yml

```diff
@@ -86,7 +91,7 @@ jobs:
         #Full path with package name, version and organization
         NPM_PKG_PATH=build/lib/$NPM_PKG_NAME/labextension/swan-cern-$NPM_PKG_NAME-$NPM_PKG_VERSION.tgz
 
-        npm publish $NPM_PKG_PATH --access public
+        npm publish $NPM_PKG_PATH --access public --dry-run
```

Comment:
@diocas 
with --dry-run the package will not be uploaded

Reply:
on sparkmonitor you have README.md and README copy.md

Reply:
> Temporary: disable pushing to NPM for now since our JupyterLab extensions are empty

Reply:
ups, I'll remove the copy..